### ISSUE TITLE
Update Sponsorship Tiers/ Packet Link

### ISSUE THREAD

**Issue by ay1741** [RIT-EVT/RIT-EVT.github.io#135]
I attached the new sponsorship packet which includes a new tier system that should be updated on the main sponsorship page as well.
[4.10 spons pack.pdf](https://github.com/RIT-EVT/RIT-EVT.github.io/files/2037685/4.10.spons.pack.pdf)

**Comment by ay1741** [comment]
Also update the title for the packet to be 2018-2019

**Comment by RyanThorna** [comment]
Yeah lol I noticed that. Wheeler helped get my local git repo squared away today so I'll get it done this week

**Comment by RyanThorna** [comment]
Hence I haven't closed the issue
